src/app/modules/account/services/msp-api-account.service.ts

```diff
@@ -948,7 +961,7 @@ private convertChildFromAccountChange(from: MspPerson): AccountChangeChildType {
       };
       output.push(partial);
     });
-
+    console.log('OUTPUT' + output.length);
```

Comment:
Hi @cyrilbravo, I would say that logging any data to the console is generally bad practice for production applications. It fine for debugging and during development, but should be removed when deployed to production.

Reply:
Hi @samuel-amos, thanks for removing the console.log. I forgot to remove it when I pushed my changes. This is noted. :)